File: android/designsystem/src/main/kotlin/com/cacompte/designsystem/tokens/Radius.kt
package com.cacompte.designsystem.tokens

import androidx.compose.ui.unit.Dp
import androidx.compose.ui.unit.dp

/**
 * Rayons de bordure (charte §5.1). Règle des coins concentriques (documentaire, pas encore un
 * point d'appel dans ce module) : un élément imbriqué utilise `rayon_parent − padding`, jamais
 * une valeur fixe — calculé au point d'usage, comme côté Swift (`Radius.swift` n'a pas non plus
 * de fonction dédiée ; iOS 26+ le fait automatiquement via `.rect(corners: .concentric)`).
 */
object Radius {
    val xs: Dp = 6.dp
    val sm: Dp = 10.dp
    val md: Dp = 14.dp
    val lg: Dp = 20.dp
    val xl: Dp = 28.dp
    val full: Dp = 999.dp
}

File: apple/CaCompteKit/Sources/DesignSystem/Tokens/Radius.swift
import CoreGraphics

/// Charte graphique §5.1 — coins concentriques : un élément imbriqué utilise
/// `rayon_parent − padding`, jamais une valeur fixe (voir `.rect(corners: .concentric)`).
public enum Radius {
  public static let xs: CGFloat = 6
  public static let sm: CGFloat = 10
  public static let md: CGFloat = 14
  public static let lg: CGFloat = 20
  public static let xl: CGFloat = 28
  public static let full: CGFloat = 999
}
